react/src/util/opReturn.ts

```diff
@@ -1,20 +1,69 @@
-export const OP_RETURN_PREFIX = 'paybutton$';
+import { lib, enc } from 'crypto-js';
 
-export function stringToHex(str: string): string {
+//opReturn format is:
+//`04` - Pushdata of the Protocol Identifier, `50415900`
+//`50415900` - PayButton protocol identifier, ascii "PAY"
+//`00` - Version 0, pushed as `OP_0`
+//`09` - pushdata for the data payload, signifying this tx has 9 bytes of data
```

Comment:
is this always 9 bytes? if so I should update the spec. We can always go to another version if you have more than 9 bytes of data, but I can imagine wanting the flexibility of data as arbitrary available bytes without needing to go to another version.

Reply:
Well it should be variable given that that's the main reason for bothering to make the nonce / payment ID optional (to claw back some bytes if they really really want it).

Reply:
Sorry, I copied your example (made some changes) but forgot to paste the example itself and... the word 'example'.

I thought the "signifying" implied the variability, but it's very subtle indeed.

Redid the text explanation; moved it to the top of the main function at the bottom.